httputil/transport.go

```diff
@@ -37,5 +38,6 @@ func newDefaultTransport() *http.Transport {
 		IdleConnTimeout:       90 * time.Second,
```

Comment:
could you change the comment before this, adding:

// but we also set ResponseHeaderTimeout which is not set by default